packages/winpanda/extra/src/winpanda/tests/unit/test_core_utils.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import mock
+
+from core import utils
+from pathlib import Path
+from core.exceptions import RCError
+
+
+def test_rc_load_string_should_fail():
+    """Check error validation when path not found."""
+    with pytest.raises(AssertionError):
+        utils.rc_load_json('')
+
+
+@mock.patch('core.utils.Path.is_absolute', return_value=True)
+def test_rc_load_absolute_path_should_fail(*args):
+    """Check error validation when path is absolute."""
+    with pytest.raises(RCError):
+        utils.rc_load_json(Path())
+
+
+@mock.patch('core.utils.Path.is_absolute', return_value=True)
+def test_rc_load_empty_content_should_fail(*args):
```

Comment:
Again this is a type check that could be eliminated using mypy.

Reply:
Agree